metadata-ws/src/test/java/uk/ac/ebi/ampt2d/metadata/cucumber/TaxonomySteps.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.cucumber;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import cucumber.api.java.en.When;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.List;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+
+public class TaxonomySteps {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @When("^user request POST taxonomies (\\d*) for id, (.*) for name and (.*) for ancestors")
+    public void performPostOnResourceUriWithStringDataAndLink(long id, String name, String linkedObjectUrlKeys) throws Exception {
```

Comment:
Can you rename the method and arguments to better reflect what this method is about?